### PR TITLE
Use new styles in all for elements

### PR DESCRIPTION
**Pull request by mrtnzlml**

### CODE REVIEW

File: app/shared/src/forms/TextInput.js [hunk 1]
```diff
@@ -2,17 +2,84 @@
...
+  onChangeText?: (text: string) => void,
+  keyboardType?: 'email-address',
+  secureTextEntry?: boolean,
+|};
+
+type State = {|
+  displayPlaceholder: boolean,
+|};
+
+export default class TextInout extends React.Component<Props, State> {
```
Comment (mrtnzlml) [inline app/shared/src/forms/TextInput.js]: I rewrote this component so we can use `Translation` in the `placeholder` prop.

Comment (tbergq) [reply app/shared/src/forms/TextInput.js]: Do you think you could also rename it to `TextInput`?

Comment (mrtnzlml) [reply app/shared/src/forms/TextInput.js]: Yes, this is not intentional... :)

File: app/shared/src/forms/TextInput.js [hunk 2]
```diff
@@ -36,26 +102,17 @@ const styles = StyleSheet.create({
     marginLeft: 10,
     alignSelf: 'center',
   },
+  placeholder: {
+    position: 'absolute',
+    padding: 10,
+    paddingVertical: 15,
+    // this should be the same with 'input' above
+    color: Color.textLight,
```
Comment (tbergq) [inline app/shared/src/forms/TextInput.js]: I don't understand this comment, color should be the same as input above? But it is not, it is textLight and input has textDark

File: app/hotels/src/filter/FilterButton.js [hunk 3]
```diff
@@ -12,46 +12,58 @@ type Props = {
...
+    });
+  }
+
+  return (
+    <View style={filterButtonStyles.wrapper}>
+      <Button
+        title={title}
+        onPress={onPress}
+        icon={icon}
+        styles={buttonStyles}
```
Comment (mrtnzlml) [inline app/hotels/src/filter/FilterButton.js]: Refactor of the buttons styles will follow.

File: app/shared/src/forms/TextInput.js (hunk 2 above)
Comment (mrtnzlml) [reply app/shared/src/forms/TextInput.js]: This should be resolved now. Please check again.

**Pull request merged**